.env

```diff
@@ -0,0 +1,21 @@
+# Do not place any sensitive credentials in this file
```

Comment:
should we tell people to create their local files and add those names to gitignore?